3ds Max/Max2Babylon/Exporter/BabylonExporter.cs

```diff
@@ -179,19 +179,28 @@ public void FlattenHierarchy(IINode node)
             }
         }
 
-        public void Export(ExportParameters exportParameters)
+        public void ExportClosedContainers()
+        {
+            List<IIContainerObject> sceneContainers = Tools.GetAllContainers();
+            foreach (IIContainerObject containerObject in sceneContainers)
+            {
+                if (!containerObject.IsInherited)continue;
+                bool merge = containerObject.MergeSource;
+            }
+            AnimationGroupList.LoadDataFromContainers();
+        }
+
+        public void Export(MaxExportParameters exportParameters)
         {
             this.exportParameters = exportParameters;
             IINode exportNode = null;
             if (exportParameters is MaxExportParameters)
```

Comment:
If we're changing the header to use the derived class, then remove this check as well.

I originally opted to use the base ExportParameters class for Export because a goal of #537 was to have both Max and Maya share the same exporter form.

Reply:
ok no problem